docs/_config.yml

```diff
@@ -55,7 +55,7 @@ JB :
   #   - Only the following values are falsy: ["", null, false]
   #   - When setting BASE_PATH it must be a valid url.
   #     This means always setting the protocol (http|https) or prefixing with "/"
-  BASE_PATH : false
+  BASE_PATH : /docs/0.6.0-incubating-SNAPSHOT
```

Comment:
is there a way to extract out the version? maybe that would be useful to show it on the doc page header like here: http://spark.apache.org/docs/latest/programming-guide.html